_pytest/capture.py

```diff
@@ -681,3 +681,14 @@ def _reopen_stdio(f, mode):
     sys.__stdin__ = sys.stdin = _reopen_stdio(sys.stdin, 'rb')
     sys.__stdout__ = sys.stdout = _reopen_stdio(sys.stdout, 'wb')
     sys.__stderr__ = sys.stderr = _reopen_stdio(sys.stderr, 'wb')
+
+
+def _attempt_to_close_capture_file(f):
+    """Suppress IOError when closing the temporary file used for capturing streams in py27 (#2370)"""
+    if six.PY2:
+        try:
+            f.close()
+        except IOError:
```

Comment:
Decided to be conservative and only ignore `IOError` under Python 2, but if others think otherwise we can ignore the exception always.

Reply:
i like it ^^